src/reviewRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -0,0 +1,138 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+/**
+ * @author Ansul Sharma <ansultaotesting.com>
+ */
+import $ from 'jquery';
+import _ from 'lodash';
+import containerHelper from 'taoQtiItem/qtiCommonRenderer/helpers/container';
+import mediaInteraction from 'taoQtiItem/qtiCommonRenderer/renderers/interactions/MediaInteraction';
+import mediaplayer from 'ui/mediaplayer';
+
+const defaults = {
+    type: 'video/mp4',
+    video: {
+        height: 270
+    }
+};
+
+/**
+ * Init rendering, called after template injected into the DOM
+ * All options are listed in the QTI v2.1 information model:
+ * http://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element10391
+ *
+ * @param {object} interaction
+ * @returns {Promise<any>}
+ */
+const render = function render(interaction) {
+    return new Promise((resolve) => {
+        const $container = containerHelper.get(interaction);
+        const media = interaction.object;
+        const $item = $container.parents('.qti-item');
+        const url = media.attr('data') || '';
+
+        /**
+         * Resize video player elements to fit container size
+         * @param {Object} mediaElement - player instance
+         * @param {jQueryElement} $container   - container element to adapt
+         */
+        const resize = _.debounce(() => {
+            let width, height;
+            if (interaction.mediaElement) {
+                height = $container.find('.media-container').height();
+                width = $container.find('.media-container').width();
+
+                interaction.mediaElement.resize(width, height);
+            }
+        }, 200);
+
+        //intialize the player if not yet done
+        const initMediaPlayer = () => {
+            if (!interaction.mediaElement) {
+                interaction.mediaElement = mediaplayer({
+                    url: url && this.resolveUrl(url),
+                    type: media.attr('type') || defaults.type,
+                    canPause: true,
+                    canSeek: true,
+                    width: media.attr('width'),
+                    height: media.attr('height'),
+                    autoStart: false,
+                    loop: false,
+                    renderTo: $('.media-container', $container)
+                })
+                    .on('render', () => {
+                        resize();
+
+                        $(window)
+                            .off('resize.mediaInteraction')
+                            .on('resize.mediaInteraction', resize);
```

Comment:
Should we unsubscribe `resize` function from window on element `destroy` ?

Reply:
There is a `resize.video` unsubscription in `destroy` method but IDK how that's different from `resize.mediaInteraction`

I don't really wanna change that part of media interactions in this PR. This PR is just concerned with overriding settings in `reviewRenderer` for scorer.